### PR TITLE
Dr/check new output

### PR DESCRIPTION
**Pull request by ravil-mobile**

### CODE REVIEW

File: src/DynamicRupture/Output/ReceiverBasedOutput.cpp [hunk 1]
```diff
@@ -75,6 +79,13 @@ void ReceiverBasedOutput::calcFaultOutput(const OutputType type,
     auto* initStress = initStresses[local.ltsId][local.nearestGpIndex];
 
     local.frictionCoefficient = (local.layer->var(drDescr->mu))[local.ltsId][local.nearestGpIndex];
+    if (containsStateVariable) {
+      auto* descr = reinterpret_cast<seissol::initializers::LTS_RateAndState*>(drDescr);
+      local.stateVariable =
+          (local.layer->var(descr->stateVariable))[local.ltsId][local.nearestGpIndex];
+    } else {
+      local.stateVariable = 0.0;
+    }
```
Comment (ravil-mobile) [inline src/DynamicRupture/Output/ReceiverBasedOutput.cpp:88]: Maybe it is better to use virtual methods

**Comment by ravil-mobile** [comment]
All tests didn't fail. I guess I can completely switch the Fortran output. What do you think?

File: src/DynamicRupture/Output/ReceiverBasedOutput.cpp (hunk 1 above)
Comment (ravil-mobile) [reply src/DynamicRupture/Output/ReceiverBasedOutput.cpp]: done

**Review by sebwolf-de — APPROVED** [review]

File: src/ResultWriter/ini_faultoutput.f90 [hunk 2]
```diff
@@ -344,6 +344,8 @@ SUBROUTINE ini_fault_receiver(EQN,MESH,BND,DISC,IO,MPI)
     INTENT(INOUT)           :: DISC
     !-----------------------------------------------------------!
 
+    ! DEPRECATED: can be removed
+#if 0
```
Comment (Thomas-Ulrich) [inline src/ResultWriter/ini_faultoutput.f90:348]: Why don't you remove it now?

**Pull request merged**